modules/application/src/main/java/edu/northeastern/truthtree/controller/stories/Stories.java

```diff
@@ -51,7 +49,7 @@ public void changeStatus(@PathVariable StoryStatus status, @PathVariable String
 
   @RequestMapping(value = "/api/stories", method = RequestMethod.PUT)
   public StoryDTO updateVotes(@RequestBody StoryDTO storyDTO,
-      @RequestParam(value = "type", required = true) VoteType type) {
+                              @RequestParam(value = "type", required = true) VoteType type) {
```

Comment:
seems like spaces are used for formatting and not formatted automatically. This however is a code formatting preference and styling. Feel free to ignore. Approving for now.